README.md

```diff
@@ -146,9 +240,7 @@ WaEnableYV12BugFixInHalfSliceChicken7 [commit 0b71cea29fc29bbd8e9dd9c641fee6bd75
    * APL, KBL: starting from kernel 4.11
    * CFL: starting from kernel 4.15
```

Comment:
I suggest to add: "ICL support is preliminary and may not be fully functional" as a separate paragraph. i915.alpha_support is not yet removed from i915 driver...

Reply:
ok, thanks for suggestion